addons/website_slides/controllers/main.py

```diff
@@ -908,11 +909,13 @@ def slide_preview(self, slide_id):
             slide.is_preview = not slide.is_preview
         return slide.is_preview
 
-    @http.route(['/slides/slide/send_share_email'], type='json', auth='user', website=True)
-    def slide_send_share_email(self, slide_id, email, fullscreen=False):
-        slide = request.env['slide.slide'].browse(int(slide_id))
-        result = slide._send_share_email(email, fullscreen)
-        return result
+    @http.route(['/slides/send_share_email'], type='json', auth='user', website=True)
+    def slide_send_share_email(self, emails, res_id, res_model, fullscreen=False):
+        for email in emails.split(','):
```

Comment:
can you check that res_model is either "slide.slide" either "slide.channel" ? (if it's not the case, you can raise a NotFound error)